nym-node/nym-node-http-api/src/state/metrics.rs

```diff
@@ -154,22 +154,30 @@ impl SharedSessionStats {
     }
 }
 
+type FinishedSessions = Vec<(u64, String)>;
+
 #[derive(Debug, Clone)]
 pub struct SessionStatsState {
     pub update_time: Date,
     pub unique_active_users: u32,
     pub session_started: u32,
-    pub session_durations: Vec<u64>,
+    pub sessions: FinishedSessions,
```

Comment:
So it's my understanding that these get purged nightly?

Reply:
Finished sessions get purged nightly yes. Each night, finished sessions of the day get published on the self-described API and purged from the Collector state. Finished sessions that were already published get lost to history. 
Active sessions are not purged, they turn into Finished Sessions when clients disconnects